packages/code-connect/components/CodeEditor/CodeEditor.figma.tsx

```diff
@@ -0,0 +1,30 @@
+import figma from '@figma/code-connect';
+import { CodeEditor } from '@patternfly/react-code-editor';
+
+/**
+ * PatternFly CodeEditor component integration for Figma Code Connect
+ */
+
+figma.connect(
+  CodeEditor,
+  'https://www.figma.com/design/aEBBvq0J3EPXxHvv6WgDx9/PatternFly-6--Components-Test?node-id=16994-184502&m=dev',
+  {
+    props: {
+      // boolean
+      showsLineNumbers: figma.boolean('Show Line Numbers'),
+
+      // enum
+      isEditable: figma.enum('Type', { Editable: true }),
+      isEmptyState: figma.enum('Type', { 'Empty state': true }),
+      isMinimapVisible: figma.enum('Type', { 'Mini Map': true })
+    },
+    example: (props) => (
+      <CodeEditor
+        emptyState={props.isEmptyState}
```

Comment:
`emptyState` is not a boolean prop. It expects you to pass it a whole EmptyState component to display when there is no code in the `code` prop.

`isEditable` is not a prop. I think you mean `isReadOnly` and it'd take the inverse of the isEditable figma prop.

Reply:
Also curious if 'isLanguageLabelVisible' is a relevant prop to any figma features?